api/types/role.go

```diff
@@ -578,18 +578,6 @@ func (r *RoleV3) CheckAndSetDefaults() error {
 		}
 	}
 
-	if r.Spec.Allow.AppLabels == nil {
```

Comment:
What about NodeLabels? Do we also want to default to nothing there as well, or would it break any use-cases?

Reply:
I've removed the wildcards for NodeLabels as well in the latest rev. It doesn't break any of the tests, and I don't think it would break any usecases.